### PR TITLE
Planetary Lighting Colors

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
<details><summary>screencaps</summary>
<p>

dreamseeker_4Gp6mnCDNE
dreamseeker_ESzoX9zGLQ
dreamseeker_0ew77loMC0
dreamseeker_bZA16gvMeA
dreamseeker_V7WYCRWN4v
dreamseeker_iTCBsMqMuL
dreamseeker_FGH6nwSdCk

</p>
</details> 

Adds some planetary lighting defines and applies them to relevant planets. I had noticed I had an old pr that did this (among other things) so I ripped this out.
Values are entirely subject to change I don't remember where I got them from and I'm not a so called 'colour theorist'

As a coincidence planetary turf files are cleaned up a little by this.

## Why It's Good For The Game
A little bit of atmosphere fluff.

## Changelog

:cl:
add: Some planets now have new lighting values. Please report back on if they are "Pleasant"
/:cl:

### CODE REVIEW

**Comment by Erikafox** [comment]
guh??

**Review by FalloutFalcon — APPROVED** [review]

**Review by FalloutFalcon — CHANGES_REQUESTED** [review]
looks very pretty but has some linters to sort out

File: code/game/objects/effects/glowshroom.dm [hunk 1]
```diff
@@ -16,8 +16,7 @@
 	var/spreadIntoAdjacentChance = 60
 	var/obj/item/seeds/myseed = /obj/item/seeds/glowshroom
 	var/static/list/blacklisted_glowshroom_turfs = typecacheof(list(
-	/turf/open/lava,
-	/turf/open/floor/plating/beach/water))
+	/turf/open/lava,))
```
Comment (FalloutFalcon) [inline code/game/objects/effects/glowshroom.dm]:
```suggestion
		/turf/open/lava
	))
```

**Review by FalloutFalcon — COMMENTED** [review]
otherwise looks good

**Review by FalloutFalcon — APPROVED** [review]

**Review by Sun-Soaked — DISMISSED** [review]
code looks fine. I'd ask spriters about doing this, though.

**Comment by Erikafox** [comment]
fuck it

**Comment by Erikafox** [comment]
ill fix it later.

**Review by Sun-Soaked — APPROVED** [review]
we'll give it a try! may end up needing to tweak some of the stronger ones, but for now it's good

**Pull request merged**